src/Sulu/Bundle/WebsiteBundle/Routing/ContentRouteProvider.php

```diff
@@ -282,4 +291,24 @@ private function decodePathInfo($pathInfo)
 
         return '/' . ltrim(rawurldecode($pathInfo), '/');
     }
+
+    /**
+     * @param RouteCollection $collection
+     */
+    public function setRouteCollection(RouteCollection $collection)
```

Comment:
Same as below, would even prefer reflections in the test if this is the only reason.